src/main/java/uk/gov/ons/ctp/response/casesvc/definition/CaseCreation.java

```diff
@@ -57,9 +59,9 @@
 
   @XmlElement(required = true)
   protected String sampleUnitType;
-
-  @XmlElement(required = true)
+  @XmlElement(nillable = true)
```

Comment:
Can you tidy up the spacing?  It's a bit hard to read that actionplanid is nillable.  There should also be a space below that for the activeenrolment variable too

Reply:
👍